visitor-app/src/shared/components/form/base.js

```diff
@@ -74,7 +74,8 @@ export const Button = styled.button`
   letter-spacing: 0.2em;
 `;
 
-export const PrimaryButton = styled(Button)`
+const standardPrimaryButtonStyling = `
+  cursor: pointer;
```

Comment:
Let's promote this into `Button`? Seems like it should be standard across all buttons.